src/gt4py/stencil_object.py

```diff
@@ -149,7 +149,7 @@ def _get_max_domain(self, field_args, origin):
                 storage_mask = tuple(field.mask)
                 if storage_mask != api_mask:
                     raise ValueError(
-                        f"The storage for '{name}' has mask '{storage_mask}', but the API signature expects '{api_mask}'"
+                        f"Storage for '{name}' has mask '{storage_mask}', API signature expects '{api_mask}'"
```

Comment:
nit: I believe this is a comma splice. The changes makes it a run-on sentence in English, because there is no conjunction connecting the `API signature expects` to the previous thought.

Reply:
There are many more of these in the PR as well.

Reply:
👍